poms/bom/pom.xml

```diff
@@ -2085,6 +2085,31 @@
                 <artifactId>javassist</artifactId>
                 <version>${javassist.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.mongodb</groupId>
+                <artifactId>bson</artifactId>
+                <version>${mongodb.version}</version>
```

Comment:
Just curious - how do these overrides work out in the Quarkus Universe BOM?

Do we end up overriding their Mongo version? Or does their version take precedence again?

Reply:
Good point I should have documented my intention. Actually the main problem is that Quarkus does not manage `bson`. That causes conflicts on our side because camel-mongodb pulls a different version transitively. Best would be to let Quarkus manage `bson` too.

Reply:
I'd improve it in a followup if you do not mind.

Reply:
Filed https://github.com/quarkusio/quarkus/issues/9330